Library/Homebrew/compilers.rb

```diff
@@ -6,6 +6,7 @@ module CompilerConstants
     "gcc-4.0"  => :gcc_4_0,
     "gcc-4.2"  => :gcc,
     "clang"    => :clang,
+    "llvm_clang" => :brew_clang,
```

Comment:
I'm somewhat surprised that sed didn't catch that, especially since the symbol in `COMPILER_PRIORITY` got changed. In any case, fixed!